sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -1204,8 +1204,27 @@ case class GpuStddevPop(child: Expression, nullOnDivideByZero: Boolean)
   override def prettyName: String = "stddev_pop"
 }
 
+case class WindowStddevSamp(
```

Comment:
Are we going to support `WindowStddevPop` too?

Reply:
Not in this PR

Reply:
Why not? The `std` family can be supported all at once with very little code added. Otherwise, we may have new requests from customers to go back, read the code, add more code + tests etc... in a significantly much more time.

IMO adding all `stddev_pop/samp` and `var_pop/samp` at once is the most optimal way.

Reply:
It might be very easy to do but that doesn't mean that we have to bundle them together. This PR is specific to a client's request and is scheduled for this release.